src/registry/variants.ts

```diff
@@ -4,12 +4,14 @@
  * Licensed under the BSD 3-Clause license.
  * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
-import * as fs from 'node:fs';
-import * as path from 'node:path';
 import { Logger, SfProject, SfProjectJson, Lifecycle } from '@salesforce/core';
 import { deepFreeze } from '../utils/collections';
 import { MetadataRegistry } from './types';
 import * as registryData from './metadataRegistry.json';
+import decomposeCustomLabelsBeta from './presets/decomposeCustomLabelsBeta.json';
```

Comment:
I do not use asynchronous imports for the presets jsons. Make the imports async will results in making all outer functions async.

Reply:
why wouldn't you use the `import * as ` and `as MetadataRegistry` pattern that works for `metadataRegistry.json` to make a Map<string, MetadataRegistry>

turning the json into a string and then turning it back into json seems odd...is that something necessary for the bundling to work?  Why is it not necessary for `metadataRegistry.json`

Reply:
I tried to compare the content
``` typescript
import * as path from 'node:path';
import * as fs from 'node:fs';
const pathToCheck = path.join(__dirname, 'presets', 'decomposeCustomLabelsBeta.json');
console.log(fs.readFileSync(pathToCheck, 'utf-8'))
import * as decomposeCustomLabelsBeta from './presets/decomposeCustomLabelsBeta.json';
console.log(JSON.stringify(decomposeCustomLabelsBeta));
```

And noticed the output of `JSON.stringify(decomposeCustomLabelsBeta)` is not what I want. 

There is a dummy default field there, which makes it different from the output of `fs.readFileSync(pathToCheck, 'utf-8')`

``` json
{"childTypes":{"customlabel":"customlabels"},"suffixes":{"label":"customlabel"},"strictDirectoryNames":{"labels":"customlabels"},"types":{"customlabels":{"children":{"suffixes":{"label":"customlabel"},"types":{"customlabel":{"directoryName":"test","id":"customlabel","name":"CustomLabel","suffix":"label","isAddressable":false,"supportsWildcardAndName":true,"uniqueIdElement":"fullName","xmlElementName":"labels"}}},"strictDirectoryName":true,"directoryName":"labels","id":"customlabels","ignoreParsedFullName":false,"name":"CustomLabels","strategies":{"adapter":"decomposed","decomposition":"topLevel","transformer":"decomposed"},"suffix":"labels","supportsPartialDelete":true}},"default":{"childTypes":{"customlabel":"customlabels"},"suffixes":{"label":"customlabel"},"strictDirectoryNames":{"labels":"customlabels"},"types":{"customlabels":{"children":{"suffixes":{"label":"customlabel"},"types":{"customlabel":{"directoryName":"test","id":"customlabel","name":"CustomLabel","suffix":"label","isAddressable":false,"supportsWildcardAndName":true,"uniqueIdElement":"fullName","xmlElementName":"labels"}}},"strictDirectoryName":true,"directoryName":"labels","id":"customlabels","ignoreParsedFullName":false,"name":"CustomLabels","strategies":{"adapter":"decomposed","decomposition":"topLevel","transformer":"decomposed"},"suffix":"labels","supportsPartialDelete":true}}}}
```

Reply:
that's an artifact of import/export (like `export default foo` it's there for you in case you're importing it not using the `*`)